pkgs/development/mobile/androidenv/cmdline-tools.nix

```diff
@@ -0,0 +1,44 @@
+{deployAndroidPackage, lib, package, autoPatchelfHook, makeWrapper, os, pkgs, pkgsi686Linux, stdenv, cmdLineToolsVersion, postInstall}:
+
+deployAndroidPackage {
+  name = "androidsdk";
+  inherit package os;
+  nativeBuildInputs = [ makeWrapper ]
+    ++ lib.optionals stdenv.isLinux [ autoPatchelfHook ];
+
+  patchInstructions = ''
+    ${lib.optionalString (os == "linux") ''
+      # Auto patch all binaries
+      autoPatchelf .
+    ''}
+
+    # Strip double dots from the root path
+    export ANDROID_SDK_ROOT="$out/libexec/android-sdk"
+
+    # Wrap all scripts that require JAVA_HOME
+    for i in $ANDROID_SDK_ROOT/cmdline-tools/${cmdLineToolsVersion}/bin
```

Comment:
`for i in $ANDROID_SDK_ROOT/cmdline-tools/${cmdLineToolsVersion}/bin; do`

Reply:
Fixed on [41acb57](https://github.com/NixOS/nixpkgs/pull/208793/commits/41acb572e21fd2d1776a0e148d059a2188d68dad)